app/src/test/kotlin/no/nav/arbeidsgiver/notifikasjon/statistikk/StatistikkModelTests.kt

```diff
@@ -196,4 +256,56 @@ class StatistikkModelTests : DescribeSpec({
             model.oppdaterModellEtterHendelse(hendelse, metadata)
         }
     }
+
+    describe("atomisk sletting for SoftDelete"){
```

Comment:
disse testene kunne vært litt mer lesbare dersom man legger litt menneskelig lesbar kontekst via `describe` og `it`

f.eks: 
```
describe("SoftDelete av sak med tilhørende notifikasjoner"){ // sier noe om hva contexten er man tester
...
it("notifikasjoner som tilhører slettet sak er også slettet") { // sier hva man forventer / en lesbar beskrivelse av assertions som er i blokken
  val notifikasjoner = ...
  notifikasjoner shouldContainExactlyInAnyOrder listOf(uuid("2"), uuid("3"))
}
```